sources/stylesheets/_fonts.scss

```diff
@@ -0,0 +1,530 @@
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Playfair Display';
+  src: url('../assets/playfair-display-v30-vietnamese_latin-ext_cyrillic-700.woff') format('woff');
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: 'Playfair Display';
+  src: url('../assets/playfair-display-v30-vietnamese_latin-ext_cyrillic-700italic.woff') format('woff');
+  font-weight: bold;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'PT Sans';
+  src: url('../assets/pt-sans-v17-latin-ext_cyrillic-ext_cyrillic-regular.woff') format('woff');
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: 'PT Sans';
+  src: url('../assets/pt-sans-v17-latin-ext_cyrillic-ext_cyrillic-italic.woff') format('woff');
+  font-weight: normal;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'PT Sans';
+  src: url('../assets/pt-sans-v17-latin-ext_cyrillic-ext_cyrillic-700.woff') format('woff');
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: 'PT Sans';
+  src: url('../assets/pt-sans-v17-latin-ext_cyrillic-ext_cyrillic-700italic.woff') format('woff');
+  font-weight: bold;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'PT Serif';
+  src: url('../assets/pt-serif-v17-latin-ext_cyrillic-ext_cyrillic-regular.woff') format('woff');
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: 'PT Serif';
+  src: url('../assets/pt-serif-v17-latin-ext_cyrillic-ext_cyrillic-italic.woff') format('woff');
+  font-weight: normal;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'PT Serif';
+  src: url('../assets/pt-serif-v17-latin-ext_cyrillic-ext_cyrillic-700.woff') format('woff');
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: 'PT Serif';
+  src: url('../assets/pt-serif-v17-latin-ext_cyrillic-ext_cyrillic-700italic.woff') format('woff');
+  font-weight: bold;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Raleway';
+  src: url('../assets/raleway-v28-vietnamese_latin-ext_cyrillic-ext_cyrillic-regular.woff') format('woff');
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: 'Raleway';
+  src: url('../assets/raleway-v28-vietnamese_latin-ext_cyrillic-ext_cyrillic-italic.woff') format('woff');
+  font-weight: normal;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Raleway';
+  src: url('../assets/raleway-v28-vietnamese_latin-ext_cyrillic-ext_cyrillic-700.woff') format('woff');
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: 'Raleway';
+  src: url('../assets/raleway-v28-vietnamese_latin-ext_cyrillic-ext_cyrillic-700italic.woff') format('woff');
+  font-weight: bold;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Roboto';
+  src: url('../assets/roboto-v30-vietnamese_latin-ext_greek-ext_greek_cyrillic-ext_cyrillic-300.woff') format('woff');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Roboto';
+  src: url('../assets/roboto-v30-vietnamese_latin-ext_greek-ext_greek_cyrillic-ext_cyrillic-300italic.woff') format('woff');
+  font-weight: 300;
+  font-style: italic;
+}
```

Comment:
These are actually not necessary, as this design only uses 400 (`normal`) and 700 (`bold`) font weights. But nothing will break by leaving them in :)